src/Model/messages.js

```diff
@@ -0,0 +1,9 @@
+const messages = {
```

Comment:
Genellikle sabit olan veya model, şema gibi veritabanı işlerinden kullanılan değişkenler diğerleinden ayırabilmek için CamelCase stilinde isimlendirilir.

---

```suggestion
const Message = {
```

Bu değişken içinde mesajları tutan değil mesajın ta kendisi olduğu için çoğul değil tekil isimlendirme yapılmalı. Dosya isimi de aynı şekilde düzeltilmeli.

---

Alanlara varsa varsayılan değerleri verilmeli. Bu değişkende `isSystemMessage`'ın değeri `false`, diğerlerininki ise `null` olabilir.